provider/cmd/pulumi-resource-pulumiservice/schema.json

```diff
@@ -311,6 +317,10 @@
           "type": "boolean",
           "description": "Skip the default dependency installation step - use this to customize the dependency installation (e.g. if using yarn or poetry)"
         },
+        "skipIntermediateDeployments": {
+          "type": "boolean",
+          "description": "Skip duplicated queued operations (it will only execute the last deployment of the same type)"
+        },
```

Comment:
Whatever we end up calling this feature, we should try to use a consistent name. I suggested calling it "skip pending" in your other PR mostly because that's what the user's issue called it.

@meagancojocar what should we call this? "skip duplicated...", "skip intermediate...", "skip pending...", etc.? I don't have a strong opinion, I just want to make sure we use language consistent with whatever we're advertising this as.

Reply:
makes sense

Reply:
I'm good with skipIntermediateDeployments. Let's stay away from pending as we use that term for something else in the CLI, duplicated isn't exactly true as they can have different commits, which leaves intermediate as the most descriptive I think. Agree with Bryce that we should be consistent.